clients/client-python/scripts/generate_doc.py

```diff
@@ -0,0 +1,25 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import pydoc
+import os
+
+from gravitino.constants.doc import DOC_DIR
+from gravitino.constants.root import GRAVITNO_DIR, MODULE_NAME
+
+if __name__ == "__main__":
+
+    if not os.path.exists(DOC_DIR):
+        # If doc folder does not exist, create a new one
+        os.makedirs(DOC_DIR)
```

Comment:
What if the folder exists and is not empty, will you overwrite the original doc? I think we should check if the folder exists and not empty, clean it first if not empty.

Reply:
Done